### DISCUSSION TITLE
Access doesn't work on Blocks

### DISCUSSION THREAD

**Discussion by jamal7887** [payloadcms/payload#4185] [Feature Requests & Ideas]
### Link to reproduction

n/a

### Describe the Bug

Using access on a block produces an error

### To Reproduce

Use access on a block 

### Payload Version

2.0.13

### Adapters and Plugins

n/a

**Comment by tylandavis** [comment]
Blocks themselves don't have access control. A field of type `blocks` can use access control, as well as the fields within a `block` config, but the block itself does not.

**Comment by jamal7887** [comment]
is there another way to hide specific blocks in the block selector?

**Comment by JarrodMFlesch** [comment]
@jamal7887 I am going to convert this into a discussion. There is no way to do this currently so it is not an issue, but I see the value here, we would just need to think through how we want to handle this, it might be something that we add under a property like `admin.access` where the access control for the block would only run on the frontend. But the discussion is a good way to get the convo started so we can flesh out the idea more.
